react/src/js/components/Consonant/Cards/Card.jsx

```diff
@@ -552,23 +555,26 @@ const Card = (props) => {
                         endDate={endDate}
                         cardStyle={cardStyle}
                         onFocus={onFocus}
-                        title={title} />
+                        title={title}
+                        tabIndex={tabIndex}
+                        renderOverlay={renderOverlay} />
                 ))}
                 {(isThreeFourths || isDoubleWide || isFull)
                     && !renderOverlay
                     && <LinkBlocker
                         target={linkBlockerTarget}
                         link={overlay}
                         title={title}
-                        getsFocus={getsFocus}
+                        getsFocus={getsFocus || true}

```

Comment:
If this is always true, doesn't that completely defeat the purpose of “only focus certain cards”?